packages/manager/modules/pci/src/projects/new/payment/payment.routing.js

```diff
@@ -0,0 +1,64 @@
+import find from 'lodash/find';
+import get from 'lodash/get';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider
+    .state('pci.projects.new.payment', {
+      url: '/payment?mode&credit&voucher&hiPayStatus&paypalAgreementStatus',
+      onEnter: /* @ngInject */ ($transition$, $window, getStepByName) => {
+        // check for paypal response in query string
+        if ($window.location.search.indexOf('paypalAgreementStatus') > -1) {
```

Comment:
`$transition$.params().paypalAgreementStatus` ?

Reply:
Not at this point :-) Paypal returns this param into query string. What I do is converting query string param to state params